components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -1806,6 +1806,14 @@ private String analyzePromptParameter(OAuthMessage oAuthMessage, OAuth2Parameter
                 } else if (containsNone) {
                     oAuthMessage.setForceAuthenticate(false);
                     oAuthMessage.setPassiveAuthentication(true);
+                } else if ((OAuthConstants.Prompt.SELECT_ACCOUNT).equals(prompt)) {
+                    /*
+                     * IS does not currently support multiple logged-in sessions.
+                     * Therefore, gracefully handling prompt=select_account by mimicking the behaviour of prompt=none
```

Comment:
This shouldn't mimic prompt none. As it has a separate error flow when there is no logged in session. We should mimic the flow similar to when no prompt param is received.

Reply:
Its not a passive auth request.